kong/llm/drivers/azure.lua

```diff
@@ -85,21 +100,25 @@ function _M.configure_request(conf)
   local parsed_url
 
   if conf.model.options.upstream_url then
-    parsed_url = socket_url.parse(conf.model.options.upstream_url)
+    parsed_url = socket_url.parse("http://127.0.0.1:8080/jack/t")
   else
     -- azure has non-standard URL format
     local url = fmt(
       "%s%s",
       ai_shared.upstream_url_format[DRIVER_NAME]:format(conf.model.options.azure_instance, conf.model.options.azure_deployment_id),
-      ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
+          conf.model.options
+      and conf.model.options.upstream_path
```

Comment:
I notice this access pattern quite a lot. Consider abstracting this to a separate function that implements path traversal.

``` lua
-- traverse `conf.model` and return matching path or nil
get_option("options.upstream_path", conf.model)
```

Reply:
I can't for this because any driver may have completely different path formatting requirement.